### DISCUSSION TITLE
Custom SMTP in local environment?

### DISCUSSION THREAD

**Discussion by stefan-girlich** [supabase/supabase#22677] [Questions] [answered]
I am using the Supabase CLI and I want to use a custom SMTP config on my local setup.
What is the best way of doing so? Can I use environment variables as some users do with docker-compose ([src](https://github.com/orgs/supabase/discussions/19238#discussion-5893409))?

**Comment by silentworks** [comment, marked as answer]
You cannot do this with the Supabase CLI. The CLI gives you sane defaults for local development, if you should need more customization then you would have to use the docker-compose method you mentioned above.

**Reply by jonathanlal** [reply]
damn would have been pretty great to send email locally with my custom smtp config 😢
